contrib/ruby/ext/trilogy-ruby/cext.c

```diff
@@ -764,6 +764,20 @@ static VALUE rb_trilogy_more_results_exist(VALUE self)
     }
 }
 
+static VALUE rb_trilogy_abandon_results(VALUE self)
+{
+    struct trilogy_ctx *ctx = get_open_ctx(self);
+
+    int count = 0;
+
+    while(ctx->conn.server_status & TRILOGY_SERVER_STATUS_MORE_RESULTS_EXISTS) {
+        VALUE result = execute_read_query_response(ctx);
```

Comment:
I benchmarked this method with an implementation written in pure Ruby using `more_results_exist?` and `next_result` to achieve the same thing. There was no significant difference (which is mostly expected).

I'm curious if there's any optimizations we should make _knowing_ we never actually needed the `result` to be passed back. Should I free the `result` here?